test/factories/noises.rb

```diff
@@ -28,9 +28,17 @@
 
     association :user, factory: :user
 
-    provider "MyProvider"
+    provider { "MyProvider" }
     text { Faker::Lorem.sentences.join(' ') }
-    created_at Time.now
+    created_at { Time.now }
+
+    Neighborhood::DISTRICTS.each do |key, district|
```

Comment:
Unused block argument - `district`. If it's necessary, use `_` or `_district` as an argument name to indicate that it won't be used.